backend/src/main/java/com/cruru/global/AuthenticationInterceptor.java

```diff
@@ -29,8 +32,16 @@ public boolean preHandle(
         return false;
     }
 
+    private boolean isGetApplyformRequest(HttpServletRequest request) {
+        return request.getRequestURI().matches(APPLYFORM_REQUEST_URI) && isGetRequest(request);
+    }
+
+    private static boolean isGetRequest(HttpServletRequest request) {
```

Comment:
```suggestion
    private boolean isGetRequest(HttpServletRequest request) {
```
static 메서드로 선언한 이유가 있으신가용??

Reply:
`cmd + opt + m`으로 메서드 분리하다가 static이 디폴트여서 들어갔습니다 😅